CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+### CHANGELOG.md is no longer updated. Please check the [releases page](https://github.com/amplitude/Amplitude-JavaScript/releases) for up to date changes.
```

Comment:
Might be good to put a date here, like `As of September 21, 2020 CHANGELOG.md is no longer updated`

Reply:
Updated